src/js/edu-benefits/5490/config/form.js

```diff
@@ -73,7 +75,14 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
-      pages: {}
+      pages: {
+        applicantInformation: applicantInformation([
+          'relativeFullName',
```

Comment:
In both cases (5490 and 1990E) the applicant is the student, not the qualifying Veteran. The Veteran info is collected later in the form. :)